composer.json

```diff
@@ -53,34 +53,34 @@
         "php": "^7.4 || ~8.0.0 || ~8.1.0",
         "ext-mongodb": "*",
         "container-interop/container-interop": "^1.2.0",
-        "doctrine/doctrine-module": "^5.0.0",
-        "doctrine/doctrine-laminas-hydrator": "^2.2.1 || ^3.0.0",
+        "doctrine/doctrine-module": "^5.1.0",
+        "doctrine/doctrine-laminas-hydrator": "^3.0.1",
```

Comment:
What's the point in dropping ^2.2.1? That's really drastic for a bugfix release

Reply:
2.2 series will soon be unmaintained